data/mods/DinoMod/monsters/juvenile.json

```diff
@@ -223,6 +223,31 @@
     ],
     "harvest": "bird_small"
   },
+  {
+    "id": "mon_nothronychus_juvenile",
+    "type": "MONSTER",
+    "name": "green and magenta two-legged juvenile",
+    "description": "A small bipedal plant-eating dinosaur juvenile with huge shiny eyes, it could be from a number of different species.",
+    "default_faction": "herbivore_dino",
+    "copy-from": "mon_gallimimus_juvenile",
+    "color": "green_magenta",
+    "volume": "800 L",
```

Comment:
So close! The juvenile still has the volume and weight of a full grown adult. Just removing those two lines will fix

Reply:
Thanks! Removed them, sorry about missing that last night.